pkg/manifests/manifests.go

```diff
@@ -2072,6 +2078,33 @@ func (f *Factory) PrometheusOperatorUserWorkloadDeployment() (*appsv1.Deployment
 	return d, nil
 }
 
+func setTLSSecurityConfiguration(namespace string, args []string, config *APIServerConfig) []string {
+	cipherSuites := strings.Join(crypto.OpenSSLToIANACipherSuites(config.GetTLSCiphers()), ",")
+	args = setArg(namespace, args, PrometheusOperatorWebTLSCipherSuitesFlag, cipherSuites)
+
+	minTLSVersion := config.GetMinTLSVersion()
+	args = setArg(namespace, args, PrometheusOperatorWebTLSMinTLSVersionFlag, string(minTLSVersion))
+
+	return args
+}
+
+func setArg(namespace string, args []string, argName string, argValue string) []string {
```

Comment:
Is it maybe worth starting to gather the prom-op args in a map and convert them?

Reply:
That also works and could be handy in the future for setting args in bulk. Updated for a single flag for now.